lmcache/cache_engine.py

```diff
@@ -380,6 +406,49 @@ def retrieve(
 
         return ret, ret_mask
 
+    @_lmcache_nvtx_annotate
+    @torch.no_grad()
+    def lookup(
+        self,
+        tokens: torch.Tensor,
+        align_to_chunk: bool,
+    ) -> torch.Tensor:
+        """
+        Checks the existence of KV cache of the tokens from the cache engine.
+        The return should be a boolean tensor indicating the existence of 
+        the KV cache of each token.
```

Comment:
Is the returned mask always a prefix (e.g., like TTTTTTFFF)? If so, maybe explicitly say that in the docstring.